### ISSUE TITLE
Inspector app improvements

### ISSUE THREAD

**Issue by bdon** [protomaps/PMTiles#49]
- [x] SVG tile previews should be zoomable
- [x] Should be able to drill down into leaf directories
- [x] should be able to preview vector tiles in leaflet
- [x] SVG should be feature-level inspectable
- [x] map preview should be feature-level inspectable
- [x] see directory Len and header-level metadata
- [x] inspect SVG with mismatched extents
- [x] correctly read tile and map hash states
- [x] split out "individual tile inspector" to separate page
- [x] view raw zxy and TIleJSON endpoints

**Comment by bdon** [comment]
See if we can reuse https://github.com/acalcutt/maplibre-gl-inspect

**Comment by bdon** [comment]
* Display labels from TileJSON

**Comment by bdon** [comment]
I'd like to port pmtiles.io to use https://github.com/maplibre/maplibre-gl-inspect , since that's what maps.protomaps.com and Maputnik are using.

**Comment by wipfli** [comment]
Other nice feature would be to load a pmtiles url not only on click on load button, but also on press ENTER in the url text input field

**Comment by bdon** [comment]
The inspector should also accept TileJSON and full Style JSONs to display, just for convenience. Raw Z/X/Y URLs won't work since we don't have enough metadata to derive `vector_layers` for inspection.

**Comment by prusswan** [comment]
> I'd like to port pmtiles.io to use https://github.com/maplibre/maplibre-gl-inspect , since that's what maps.protomaps.com and Maputnik are using.

Actually, I think both implementations have their merits (I like how pmtiles.io works with popup freezing and treats each source layer as a group, with possibly different geometry types). For the metadata, maplibre-gl-inspect only needs a list of layers present in the source (e.g. ["water",...}]).

btw, I noticed that switching from Map View to Tile Inspector does not work for local files (you have to drag the file again)

**Comment by bdon** [comment]
Yeah, the plan is to keep tile inspector as a separate page instead of all under one single page app, because trying to manage the hash state gets too complicated.

**Comment by bdon** [comment]
It would be helpful to perform light validation of the PMTiles header/metadata to assist implementers

**Comment by bdon** [comment]
Make sure #537 is supported

**Comment by bdon** [comment]
Closing this old mega-issue in favor of a feedback issue on the new http://beta.pmtiles.io
